docs/getting_started/trivia_storage.md

```diff
@@ -58,7 +58,7 @@ storeLargePayload: (state, event) => {
 
 ## Global storage
 
-If you want to store things globally, you can use the built-in [Key-Value-Store (KVS)](../../advanced/kvs).
+If you want to store things globally, you can use the built-in [Key-Value-Store (KVS)](../../advanced/kvs). <!-- can't find in repo -->
```

Comment:
@dmytropanontko , you can add a link to https://botpress.io/docs/latest/recipes/dbs/